Fix Time.to_sec_string example

dbohdan opened the pull request:
As of Core [version 112.17.00](https://github.com/janestreet/core/commit/5ee4c65df85e5720dbbcc14087e2ef28259930ab) Time.to_sec_string requires a time zone.

dbohdan committed:
Fix Time.to_sec_string example

As of Core version 112.17.00 Time.to_sec_string requires a time zone.

zzhjerry commented on code/guided-tour/main.topscript:
```diff
@@ -147,7 +147,7 @@ let log_entry maybe_time message =
       | None -> Time.now ()
     in
-    Time.to_sec_string time ^ " -- " ^ message
+    Time.to_sec_string Time.Zone.local time ^ " -- " ^ message
```
The right sequence should be `Time.to_sec_string time Time.Zone.local`

dbohdan commented:
@zzhjerry Thanks for pointing that out! I've replaced the patch.

douglarek commented:
```
utop # Time.to_sec_string ~zone:Time.Zone.local Time.now();;
Error: This function has type Time.t -> zone:Time.Zone.t -> string It is applied to too many arguments; maybe you forgot a `;'.
```

dbohdan committed:
fix `Time.to_sec_string` example again

`Time.Zone.local` has been removed from the package `core`. The current
version, 0.16.1, has a deprecation warning in its place. There is a
platform-specific replacement in `core-unix`. To avoid requiring
`core-unix`, let's change the time zone to UTC.

dbohdan commented:
Core has undergone changes since I submitted the PR. I have updated the PR for version 0.16.1 (the version numbering scheme has changed, too).
